Source/Csla/Configuration/Fluent/DataPortalServerOptions.cs

```diff
@@ -0,0 +1,52 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="DataPortalServerOptions.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Server-side data portal options.</summary>
+//-----------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using Csla.Server;
+
+namespace Csla.Configuration
+{
+  /// <summary>
+  /// Server-side data portal options.
+  /// </summary>
+  public class DataPortalServerOptions
+  {
+    /// <summary>
+    /// Gets or sets a value containing the type of the
+    /// IDataPortalAuthorizer to be used by the data portal.
+    /// An instance of this type is created using dependency
+    /// injection.
+    /// </summary>
+    public Type AuthorizerProviderType { get; set; } = typeof(ActiveAuthorizer);
+
+    /// <summary>
+    /// Gets a list of the IInterceptDataPortal instances
+    /// that should be executed by the server-side data portal.
+    /// injection.
+    /// </summary>
+    public List<IInterceptDataPortal> InterceptorProviders { get; } = new List<IInterceptDataPortal>();
```

Comment:
This seems to accept constructed instances of interceptors. I don't think that is what we would want; instead we would want a list of types that we want to be instantiated by DI when we need them - so that they are provided any dependencies by the container at the point they are instantiated.

Reply:
I didn't _set out_ to replace your work - but I was in the thick of changing all the other code, and I thought I could save us both some effort :)